sdk/analysisservices/Azure.ResourceManager.Analysis/CHANGELOG.md

```diff
@@ -1,15 +1,12 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.1.0 (2023-02-16)
```

Comment:
if no API change in this SDK since its last release, we should use 1.0.1 instead

Reply:
and if there are some API changes in this SDK since its last release, please add some changelog entries to describe the changes